systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -234,6 +234,10 @@ public static boolean isNamespaceRbacScope() {
         return STRIMZI_RBAC_SCOPE_NAMESPACE.equals(STRIMZI_RBAC_SCOPE);
     }
 
+    public static boolean isStrimziPodSetEnabled() {
+        return STRIMZI_FEATURE_GATES.contains(Constants.USE_STRIMZI_POD_SET);
```

Comment:
I'm not sure there is a way to do this better. But once the feature gate transitions to beta and is enabled by default, this method will stop working. I'm not sure it can be done in a more resilient way, but having some comment abotu that might be helpful.

Reply:
I will think about it more. This was first and easiest way, how to do it, so I just picked it. Anyway I totally agree, this is not sufficient for the future.

Reply:
Is there an option to get it directly from CO? Maybe from some constant class, but I don't want to introduce a dependency on `cluster-operator` package, but if it's available in `operator-common` we can work with it?

Reply:
Discussed offline -> there is no easy or clear way, how to check this. The comment about this is added and we will have to keep in mind that once `StrimziPodSet` FG will be moved to beta, we need to change this.